src/main/java/org/itc/AppConfig.java

```diff
@@ -17,4 +24,17 @@ public class AppConfig {
     public String appName() {
         return env.getProperty("app.name");
     }
+
+    @Bean
+    public DataSource dataSource() {
+        return new EmbeddedDatabaseBuilder()
+                .setType(EmbeddedDatabaseType.H2)
+                .addScript("schema.sql")
+                .build();
+    }
+
+    @Bean
+    public JdbcTemplate jdbcTemplate(DataSource dataSource) {
+        return new JdbcTemplate(dataSource);
+    }
```

Comment:
Si van a trabajar con spring framework, deben definir los siguientes beans (Hacer los respectivos imports)
```suggestion
    }
    
    @Bean
    public NamedParameterJdbcOperations namedParameterJdbcOperations(DataSource dataSource) {
        return new NamedParameterJdbcTemplate(dataSource);
    }

    @Bean
    public PlatformTransactionManager transactionManager(DataSource dataSource) {
        return new DataSourceTransactionManager(dataSource);
    }
```